apps/web/screens/News.tsx

```
@@ -351,27 +351,28 @@ const NewsListNew: Screen<NewsListProps> = ({
-                  />
-                </Box>
-              )
-            })}
+            {newsList.map(
+              ({ title, intro, image, slug, date, genericTags }, index) => {
+                const mini =
+                  selectedPage > 1 || (selectedPage === 1 && index > 2)
+
+                return (
+                  <Box marginBottom={mini ? spacingMini : spacing}>
+                    <DigitalIcelandNewsCard
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)